examples/chip-tool/templates/commands.zapt

```diff
@@ -46,6 +46,13 @@ CHIP_ERROR LogValue(const char * label, size_t indent, X value)
     return CHIP_NO_ERROR;
 }
 
+template <typename X, typename std::enable_if_t<std::is_floating_point<X>::value && !std::is_same<std::remove_cv_t<std::remove_reference_t<X>>, bool>::value, int> = 0>
```

Comment:
If think if `is_floating_point<X>` then we can be sure that `X` is not bool, so the further part of the condition is not needed.

Reply:
Good catch, fixed.